src/style.rs

```diff
@@ -1,10 +1,7 @@
-use std::borrow::Cow;
-
-use console::{measure_text_width, Style};
-
 use crate::format::{BinaryBytes, DecimalBytes, FormattedDuration, HumanBytes, HumanDuration};
 use crate::progress::ProgressState;
 use crate::utils::{expand_template, pad_str};
+use console::{measure_text_width, Style};
```

Comment:
Please keep the third-party crate exports above the local crate imports, in a separate block.